packages/bpk-component-theme-toggle/src/theming.js

```diff
@@ -19,14 +19,27 @@ import {
   colorBlue500,
 } from 'bpk-tokens/tokens/base.es6';
 
+/* eslint-disable no-unused-vars */
```

Comment:
Whats the unused var here?

Reply:
The secondary ones are all unused - but I think it's good to have them in there for future usage. This is essentially acting as a mini version of `bpk-tokens` for theming variables (which we may not want inside `bpk-tokens` as they're just for usage here).

Reply:
Ah in that case it makes sense to wait until #315 because MY PR uses them. :)

Reply:
Damn, you got me!